ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sno/etcd-encryption/cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-etcd-encryption-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-etcd-encryption
```

Comment:
I think we do not need to introduce this install workflow, because of https://github.com/openshift/release/pull/49845/files#r1524364786

Reply:
removed.